FlowCrypt/src/main/java/com/flowcrypt/email/extensions/kotlin/ByteArrayExt.kt

```diff
@@ -0,0 +1,58 @@
+/*
+ * © 2016-present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com
+ * Contributors: Ivan Pizhenko
+ */
+
+package com.flowcrypt.email.extensions.kotlin
+
+fun ByteArray.toLowerHexString(): String {
+  val result = StringBuilder()
+  for (b in this) {
+    result.append("%02x".format(b))
+  }
+  return result.toString()
+}
+
+fun ByteArray.toUpperHexString(): String {
+  val result = StringBuilder()
+  for (b in this) {
+    result.append("%02X".format(b))
+  }
+  return result.toString()
+}
+
+// KMP search https://stackoverflow.com/a/67275192/1540501
+fun ByteArray.indexOf(needle: ByteArray): Int {
```

Comment:
It seems that this method is unused in the code

Reply:
Actually, all code in this file is unused. @IvanPizhenko Please clarify should we leave it as-is?

Reply:
Let's remove it